### PR TITLE
[Fused] Remove fused LeakyRelu

### PR DESCRIPTION
**Pull request by mbencer**
Fused LeakyRelu introduced in https://github.com/NervanaSystems/ngraph/pull/2919 was removed

### CODE REVIEW

File: src/ngraph/runtime/cpu/builder/leaky_relu.cpp [hunk 1]
```diff
@@ -30,21 +30,21 @@ namespace ngraph
         namespace cpu
         {
             template <>
-            void Builder::BUILDER_DECL(ngraph::op::CPULeakyRelu)
+            void Builder::BUILDER_DECL(ngraph::op::LeakyRelu)
```
Comment (postrational) [inline src/ngraph/runtime/cpu/builder/leaky_relu.cpp]: Let's keep the `ngraph::op::LeakyRelu` -> `ngraph::op::CPULeakyRelu` change.

Comment (mbencer) [reply src/ngraph/runtime/cpu/builder/leaky_relu.cpp]: Done

File: src/ngraph/runtime/cpu/op/leaky_relu.cpp [hunk 2]
```diff
@@ -20,7 +20,7 @@ using namespace std;
 using namespace ngraph;
 
 op::CPULeakyRelu::CPULeakyRelu(shared_ptr<Node> arg, float alpha)
-    : UnaryElementwiseArithmetic("CPULeakyRelu", {arg})
+    : UnaryElementwiseArithmetic("LeakyRelu", {arg})
```
Comment (postrational) [inline src/ngraph/runtime/cpu/op/leaky_relu.cpp]: We a `CPULeakyRelu` change here.

Comment (mbencer) [reply src/ngraph/runtime/cpu/op/leaky_relu.cpp]: Done

File: src/ngraph/frontend/onnx_import/op/leaky_relu.cpp [hunk 3]
```diff
@@ -42,8 +48,8 @@ namespace ngraph
                     std::shared_ptr<ngraph::Node> alpha_node =
                         std::make_shared<ngraph::op::Constant>(
                             data->get_element_type(), Shape{}, std::vector<double>{alpha});
-
-                    return {std::make_shared<ngraph::op::LeakyRelu>(data, alpha_node)};
+                    alpha_node = ngraph::op::make_broadcast_node(alpha_node, data->get_shape());
```
Comment (GlebKazantaev) [inline src/ngraph/frontend/onnx_import/op/leaky_relu.cpp]: Why not to replace it with PReLU op?

Comment (postrational) [reply src/ngraph/frontend/onnx_import/op/leaky_relu.cpp]: We already have a PReLU op:
https://github.com/NervanaSystems/ngraph/blob/b69d3fd68e6a71baaefaab9e96302915f76f7f52/src/ngraph/op/fused/prelu.hpp#L31

**Pull request merged**